proto/testupdater/test_updater.proto

```diff
@@ -16,16 +16,28 @@ syntax = "proto3";
 
 option go_package = "github.com/grpc/test-infra/proto/endpointupdater";
 
-package endpointupdater;
+package testupdater;
 
-service EndpointUpdater {
+service TestUpdater {
   // Sends an update
-  rpc UpdateEndpoint (EndpointUpdaterRequest) returns (EndpointUpdaterReply) {}
+  rpc UpdateTest (TestUpdateRequest) returns (TestUpdateReply) {}
+
+  rpc QuitTestUpdateServer(Void) returns (Void);
```

Comment:
Optional: RPCs like this at Google are traditionally called `QuitQuitQuit`

Reply:
I will update it. :)

Reply:
It's not important. Don't worry about the name change.